app/assets/stylesheets/layouts/_header.sass

```diff
@@ -18,7 +18,7 @@
   color: $white
 
 .header-divider
-  background-image: url('top-menu-colored-bar.png')
+  background-image: url('/assets/top-menu-colored-bar.png')
```

Comment:
You can use `image-url` here, to simplify. Check it out: https://github.com/rails/sass-rails#asset-helpers